### PR TITLE
Stable Goldilocks Release

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

File: stable/goldilocks/README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
-# goldilocks
+<div align="center">
+<a href="https://github.com/FairwindsOps/goldilocks"><img src="/logo.svg" height="150" alt="Goldilocks" style="padding-bottom: 20px" /></a>
```
Comment (rbren) [inline stable/goldilocks/README.md]: I think you need to remove the `/` in `/logo.svg` to reference the logo in this directory. Can also probably remove the `icon.svg` file

File: stable/goldilocks/README.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+
+This is a Helm chart for the Fairwinds [Goldilocks project](https://github.com/FairwindsOps/goldilocks). It provides an easy way to install the controller and the dashboard for viewing resource recommendations in your Kubernetes cluster. Please see the [Goldilocks README](https://github.com/FairwindsOps/goldilocks) for more information.
 
 ## Requirements
 
-This has a hard requirement on VPA being installed. Please see the [Goldilocks README](https://github.com/fairwindsops/goldilocks)
+This has a hard requirement on VPA being installed. Please see the [Goldilocks README](https://github.com/FairwindsOps/goldilocks)
 
-VPA Can be installed using a post-install hook if `installVPA` is set to true in the values.
+VPA Can be installed using a post-install hook if `installVPA` is set to true in the values. If you later wish to upgrade the vpa controller to the latest supported by this chart, you can enable the `upradeVPA` option to run the upgrade.
```
Comment (rbren) [inline stable/goldilocks/README.md]: `s/uprade/upgrade`

File: stable/goldilocks/README.md (hunk 1 above)
Comment (sudermanjr) [reply stable/goldilocks/README.md]: Fixed. Icon needs to be there for the Chart.yaml reference

File: stable/goldilocks/README.md (hunk 2 above)
Comment (sudermanjr) [reply stable/goldilocks/README.md]: Fixed. Thanks.

**Review by bambash — APPROVED** [review]
:tada: looks great! :bed: :shallow_pan_of_food: :seat:

**Pull request merged**